thirdeye/thirdeye-frontend/app/pods/components/rootcause-metrics/component.js

```diff
@@ -189,10 +251,10 @@ export default Component.extend({
    * @type {Array}
    */
   preselectedItems: computed({
-    get () {
+    get() {
       return [];
     },
-    set () {
+    set() {
       // ignore
     }
   }),
```

Comment:
it's better to add TODO and Jira Ticket to trace.

Reply:
This is a prettify of old code.  I'm not sure what TODO would go here

Reply:
We can refactor later. 
From the codebase, there are no dependent properties on the computed property. It's not appropriate here. we can be refactored when we migrate the code to the new codebase.
```
 preselectedItems: computed({
    get () {	    get() {
      return [];	      return [];
    },	    },
    set () {	    set() {
      // ignore	      // ignore
    }	    }
  }),	  }),
```

Reply:
If nothing is using this property, I'll go ahead and delete it.

Reply:
The property gets passed down, so I believe it is being used.  It seems it was done to address a bug.  I'd better leave it.  https://github.com/apache/incubator-pinot/commit/fb3c74dfd6f4c5dafdf42a11ba7977426b0dcf2a